### ISSUE TITLE
Load tool APIs when a project is selected

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#4672]
### User Story

As a developer, I want to access the tool APIs before the tool on the tool page so that I can use it to display information on the tools page.

### Features / Specifications
- [ ] Move book loading (change how books are loaded)
- [ ] This will allow the `Invalidated` to display accurate numbers
- [ ] This will allow the wA tool to load alignments for an entire language

### Definition of Done
- [ ] The tool APIs are accessible after the project has been selected.
- [ ] `Invalidated` numbers are displaying accurately. 
- [ ]

**Comment by da1nerd** [comment]
When a project is selected in tC the following must occur:
1. load the project meta data (already happening but will likely be streamlined)
2. Initialize the tool APIs (tw and wa).
3. Navigate to the tools page
4. Display correct progress and error information on each tool by reading from the tool api.
5. When a tool is selected it should open almost instantly because most of the project data will have already been loaded.

**Comment by da1nerd** [comment]
Note: eventually we'll be able to utilized these now more accessible tool APIs when importing a project. e.g. to read alignment information, validate, etc. This will reduce duplicate code and ensure consistency.

**Comment by da1nerd** [comment]
tools will actually be initialized before a project is selected.
The tools  apis will however be initialized when a project is selected.

**Comment by da1nerd** [comment]
This is basically done. Just waiting for the other pieces to get finished.

**Comment by da1nerd** [comment]
This is currently testable with wA. tW will be testable after #4884 is completed.

**Comment by cckozie** [comment]
1.1.0 (b287d3d) 
The reset alignments are handled differently when the project is imported and immediately loaded vs when it is selected and then loaded starting on the projects page.

Also, although the wA tool card shows 2 reset alignments, I can only find one in the project. That one is at verse 19:41, the missing verse. (I may have missed it when scrolling through the book in wA. Can you point me to the other verse with the broken link icon?)

When the project is imported, the reset alignments alert is displayed at the end of the validation process, but the wA tool card shows 0 broken links.
![image](https://user-images.githubusercontent.com/8171759/50600837-025ca080-0e78-11e9-924a-29e791f31854.png)

After returning to the projects page and selecting a different project, then going back to the projects page and selecting the imported project, the tool card now shows 2 broken links.
![image](https://user-images.githubusercontent.com/8171759/50600973-6aab8200-0e78-11e9-8c1d-902405b49830.png)

**Comment by da1nerd** [comment]
The problem with the invalid checks count not being displayed after an import has been fixed in https://github.com/unfoldingWord-dev/translationCore/pull/5636

As for your missing invalid verse, I can't help you with that without your project. I've tested opening projects with invalidations and have confirmed the count is reliable. Finishing https://github.com/unfoldingWord-dev/translationCore/issues/4443 will make finding these a lot easier.
